CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## 2.0.2
```

Comment:
Please, write `pending` instead of version number. You don't know which version will be the next.